certbot/docs/install.rst

```diff
@@ -78,74 +67,6 @@ choosing "snapd" in the "System" dropdown menu. (You should select "snapd"
 regardless of your operating system, as our instructions are the same across
 all systems.)
 
-.. _certbot-auto:
-
-Certbot-Auto
-------------
-
-The ``certbot-auto`` wrapper script installs Certbot, obtaining some dependencies
-from your web server OS and putting others in a python virtual environment. You can
-download and run it as follows::
-
-  wget https://dl.eff.org/certbot-auto
-  sudo mv certbot-auto /usr/local/bin/certbot-auto
-  sudo chown root /usr/local/bin/certbot-auto
-  sudo chmod 0755 /usr/local/bin/certbot-auto
-  /usr/local/bin/certbot-auto --help
-
-To remove certbot-auto, just delete it and the files it places under /opt/eff.org, along with any cronjob or systemd timer you may have created.
-
-To check the integrity of the ``certbot-auto`` script,
-you can use these steps::
-
-
-	    user@webserver:~$ wget -N https://dl.eff.org/certbot-auto.asc
-	    user@webserver:~$ gpg2 --keyserver pool.sks-keyservers.net --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
-	    user@webserver:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.asc /usr/local/bin/certbot-auto
-
-
-
-The output of the last command should look something like::
-
-
-	    gpg: Signature made Wed 02 May 2018 05:29:12 AM IST
-	    gpg:                using RSA key A2CFB51FA275A7286234E7B24D17C995CD9775F2
-	    gpg: key 4D17C995CD9775F2 marked as ultimately trusted
-	    gpg: checking the trustdb
-	    gpg: marginals needed: 3  completes needed: 1  trust model: pgp
-	    gpg: depth: 0  valid:   2  signed:   2  trust: 0-, 0q, 0n, 0m, 0f, 2u
-	    gpg: depth: 1  valid:   2  signed:   0  trust: 2-, 0q, 0n, 0m, 0f, 0u
-	    gpg: next trustdb check due at 2027-11-22
-	    gpg: Good signature from "Let's Encrypt Client Team <letsencrypt-client@eff.org>" [ultimate]
-
-
-
-The ``certbot-auto`` command updates to the latest client release automatically.
-Since ``certbot-auto`` is a wrapper to ``certbot``, it accepts exactly
-the same command line flags and arguments. For more information, see
-`Certbot command-line options <https://certbot.eff.org/docs/using.html#command-line-options>`_.
-
-For full command line help, you can type::
-
-  /usr/local/bin/certbot-auto --help all
-
-Problems with Python virtual environment
```

Comment:
Related to https://github.com/certbot/certbot/pull/8391, I wanted to note that the workaround to handle low memory system will certainly be relevant for an installation procedure that would use pip. We are planning to add this with https://github.com/certbot/certbot/issues/8422.

It may be relevant to create an issue or add it to the documentation in https://github.com/certbot/certbot/issues/8422 to remember about this, including the proposition to use `dd` instead of `fallocate` as said in https://github.com/certbot/certbot/pull/8391.

Reply:
Did you read my comment at https://github.com/certbot/certbot/pull/8532#discussion_r542948604? I argued that this is largely a historical problem, suggested we not worry about it, and we can revisit that decision if (m)any people hit it which I'm not confident they will.

Reply:
Ah sorry. Indeed I read it yesterday and forgot about it today while studying the other PR about this section.

So indeed it is not really useful anymore.